pkg/config/config.go

```diff
@@ -248,8 +269,35 @@ func CustomHTTPErrorHandler(err error, c echo.Context) {
 	}
 }
 
-func ConfigureEcho() *echo.Echo {
-	e := echo.New()
+func metricsMiddlewareSkipper(ctx echo.Context) bool {
+	path := ctx.Request().URL.Path
+	switch {
+	case path == "/ping" || path == "/ping/":
+		return true
+	case path == "/metrics" || path == "/metrics/":
+		return true
+	}
+	pathItems := handler_utils.NewPathWithString(path).RemovePrefixes()
+	if pathItems.StartWithResources(
+		[]string{"ping"},
+		[]string{"repositories", "validate"},
+		[]string{"repository_parameters", "validate"},
+	) {
```

Comment:
I was thinking about what needs to be skipped. Since the `/metrics` endpoint is on a different port, I don't think it needs to be skipped here. It would be an unregistered route on this server. In which case, only `/ping` needs to be skipped. 

Also, I think you could just hard code this and avoid the tricky path parsing.

Reply:
thanks mate! I tracked down information about the path and port for the metrics, and if it is actually hardcoded into a clowdenvironment resource, if that information change, would make wrong several things; I have to update all the matchings with '/metrics' to use the path read from configuration and the same for the metrics port.